code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
```

Comment:
what if you add the meta viewport tag? I think it can help with screen responsiveness